lib/actions/model.js

```diff
@@ -45,8 +46,14 @@ module.exports = {
       if (preSaveAction) dispatch(preSaveAction);
 
       const Model = storageManager.storage(brainstemKey).model;
-      const xhr = new Model({ id: modelId })
-        .save(attributes, saveOptions);
+      const model = new Model({ id: modelId });
+
+      if (!model.isValid()) {
+        dispatch(invalidAction(model.validationError));
+        return model.validationError;
+      }
+
+      const xhr = model.save(attributes, saveOptions);
```

Comment:
I don't think that's a good idea... Unless you meant that we should be passing `validate: false` to the action creator itself instead of hard-coding it.